src/main/java/edu/kis/powp/jobs2d/command/manager/parsing/Parser.java

```diff
@@ -0,0 +1,14 @@
+package edu.kis.powp.jobs2d.command.manager.parsing;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+
+import java.io.File;
+import java.util.List;
+
+public interface Parser {
+	List<DriverCommand> parseFromImport();
```

Comment:
The interface is inconsistent!!! It defines an export format, but no import format. Internal format for import is List of commands and for export, it is a DriverCommand.